api/scpca_portal/models/job.py

```diff
@@ -44,6 +44,10 @@ class Meta:
     # Datasets should never be deleted
     dataset = models.ForeignKey(Dataset, null=True, on_delete=models.SET_NULL, related_name="jobs")
 
+    # Maximum size of a dataset in GB in order to be accommodated by the fargate pipeline
+    # Number calculated as 2.5x size of dataset which would hit 200 GB ephemeral storage max
+    MAX_FARGATE_SIZE_IN_GB = 80
```

Comment:
```suggestion
    MAX_FARGATE_SIZE_BYTES = 80 * 1000000000
```

I think its more important to be consistent with unit vs easier to read.
Move 1000000000 to common.GB to improve readability